ansible/roles/redis/templates/systemd.j2

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Redis In-Memory Data Store
+After=network.target
+
+[Service]
+User={{ learner_user }}
+Group={{ learner_user }}
+ExecStart={{ redis_dir }}/redis-stable/src/redis-server {{ redis_dir }}/redis-stable/redis.conf
```

Comment:
@Kaali09 @gandham-santhosh - We should disable protected-mode for Redis. Please check the below task which we removed in this pull-request.

```
- name: start redis
  become: yes
  become_user: "{{learner_user}}"
  shell: nohup src/redis-server {{learner_user_home}}/redis-stable/redis.conf --protected-mode no &
  args:
    chdir: "{{learner_user_home}}/redis-stable/"
  async: 60
  poll: 60
```

Reply:
@maheshkumargangula  - We already disabled the protected-mode in the redis conf file. Again do we need to mention in the start command? 

`# By default protected mode is enabled. You should disable it only if
# you are sure you want clients from other hosts to connect to Redis
# even if no authentication is configured, nor a specific set of interfaces
# are explicitly listed using the "bind" directive.
protected-mode no`

Reply:
ok.